app/helpers/practices_helper.rb

```diff
@@ -97,4 +97,9 @@ def get_tool_resource_text(resource_type)
     }
     content[resource_type.to_sym]
   end
+
+  def sort_adoptions_by_state_and_station_name(adoptions)
```

Comment:
Question... I am wondering if we should use this same style of sorting for the adoption editor page facilities + CRH combo box because it makes more logical sense than how the options are organized now �
<img width="550" alt="Screen Shot 2022-03-02 at 10 20 52 AM" src="https://user-images.githubusercontent.com/20211771/156403143-76ae8237-eb1e-4f78-8ebf-9aec65a8eeae.png">
�